### DISCUSSION TITLE
Can I offer my help to validate applet PRs?

### DISCUSSION THREAD

**Discussion by claudiux** [linuxmint/discussions#54] [Pull requests]
I noticed that about twenty applet PRs were waiting to be validated. Can I offer to help validate them? I'll use a virtual machine (and for some a 'physical' computer, when the applet interacts with the hardware) to validate them.

**Comment by rcalixte** [comment]
Same. If there are volunteers needed to review Spices, I can offer myself as tribute as well. It would help if there were documentation or some guidelines from things like managing timelines and expectations to squashing and merging commits to keep the commit history clean. (In the past, @brownsr has said his role is to prevent malware from entering the official repos so that is a minimum of expectations in code review.)

**Comment by clefebvre** [comment]
Thanks @claudiux and @rcalixte. Please drop us an email at root@linuxmint.com so we can get in touch.
